README.md

```diff
@@ -56,43 +83,35 @@ A utility is provided to run _clean, build, and test_ in sequence with:
 mage cbt
 ```
 
```

Comment:
Why remove the pointer to CI? Would be great for folks to have a good pointer to where jobs are run and the build history incl logs. More appropriate likely in `CONTRIBUTING` rather than the `README`, I agree; how about verifying these links are right and moving this section there?

Same can be said then for the "Building" and "Deployment" sections too - `CONTRIBUTING` for devs of this project, `README` for consumers.

Reply:
I don't know what happened here, i didnt mean to delete this or what this is

Reply:
this wasn't deleted in the code I still have ...

```A utility is provided to run _clean, build, and test_ in sequence with:

```
mage cbt
```

Reply:
Looks either resolved now *or* IDK what I could have possibly been talking about; there are no diffs on this line in the PR now.